pkgs/applications/emulators/libretro/cores/mupen64plus.nix

```
@@ -20,6 +21,26 @@ mkLibretroCore {
     hash = "sha256-nII/PMYo2xLznmAcKs6jDWGRS1DC3tiDeT6KJKRnaCI=";
   };
 
+  # Fix for GCC 14
+  # https://github.com/libretro/mupen64plus-libretro-nx/pull/526
+  patches = [
+    (fetchpatch2 {
+      name = "minizip-avoid_trying_to_compile_problematic_code.patch";
+      url = "https://github.com/libretro/mupen64plus-libretro-nx/commit/2b05477dd9cd99e7f9425f58cb544f454fc0d813.patch";
```

fetchpatch2 needs to add `?full_index=1` to the end of the url: https://github.com/NixOS/nixpkgs/issues/257446
```suggestion
      url = "https://github.com/libretro/mupen64plus-libretro-nx/commit/2b05477dd9cd99e7f9425f58cb544f454fc0d813.patch?full_index=1";
```

---

Done.